### PR TITLE
Add chromium path arguments

### PR DESCRIPTION
**Pull request by RodBerw**

### CODE REVIEW

File: bin/simulate [hunk 1]
```diff
@@ -35,7 +35,7 @@ function unknownOptHandler(optName) {
 function processArgs() {
     try {
         var parserOptions = {
-            string: ['dir', 'simhostui', 'target', 'simulationpath', 'device', 'lang', 'theme', 'middleware'],
+            string: ['dir', 'simhostui', 'target', 'simulationpath', 'device', 'lang', 'theme', 'middleware', 'chromiumPath'],
```
Comment (EzioLi01) [inline bin/simulate:38]: Using lowercase for argument.

File: src/browsers/browser.js [hunk 2]
```diff
@@ -73,6 +74,7 @@ function getBrowser(target, dataDir, url) {
     const browsers = {
         win32: {
             chrome: `chrome --user-data-dir=%TEMP%\\${dataDir}`,
+            chromium: chromiumPath || 'chromium',
```
Comment (EzioLi01) [inline src/browsers/browser.js:77]: Only path is enough for windows I think.

**Comment by EzioLi01** [comment]
Generally I think chromium can only support windows here. You can add some logs:
1. In win32 but not path argument set, will throw exception
2. In other OSs and set path, show log to notice that not support.

File: bin/simulate [hunk 3]
```diff
@@ -77,7 +77,8 @@ function processArgs() {
             middleware: parsed.middleware,
             generateids: parsed.generateids,
             spaurlrewrites: parsed.spaurlrewrites,
-            showbrowser: parsed.showbrowser
+            showbrowser: parsed.showbrowser,
+            chromiumPath: parsed.chromiumPath
```
Comment (EzioLi01) [inline bin/simulate:81]: Also lowercase?

File: bin/simulate [hunk 4]
```diff
@@ -102,7 +103,8 @@ function getUsageString() {
         '[--middleware MIDDLEWARE_PATH]',
         '[--generateids (GENERATEIDS)]',
         '[--spaurlrewrites (SPA_URL_REWRITES)]',
-        '[--showbrowser (SHOWBROWSER)]'
+        '[--showbrowser (SHOWBROWSER)]',
+        '[--chromiumPath CHROMIUM_PATH]'
```
Comment (EzioLi01) [inline bin/simulate:107]: lowercase

File: bin/simulate [hunk 5]
```diff
@@ -77,8 +77,10 @@ function processArgs() {
             middleware: parsed.middleware,
             generateids: parsed.generateids,
             spaurlrewrites: parsed.spaurlrewrites,
-            showbrowser: parsed.showbrowser
+            showbrowser: parsed.showbrowser,
+            chromiumpath: parsed.chromiumpath
         };
+        console.log(chromiumpath)
```
Comment (EzioLi01) [inline bin/simulate:83]: Remove this log.

File: src/simulate.js [hunk 6]
```diff
@@ -17,18 +17,27 @@ var simulate = function (opts) {
...
     var simulator = new Simulator(opts);
     var showBrowser = opts.showbrowser;
+    var chromiumPath = opts.chromiumpath;
 
     if (!showBrowser) {
         var noBrowserMessage = 'The argument `showbrowser` is set to false. Please load simulated application in browser manually if needed.';
         log.warning(noBrowserMessage);
     }
 
+    if(process.platform === 'win32' && !chromiumPath){
```
Comment (EzioLi01) [inline src/simulate.js:27]: We have several browsers as the target, only chromium needs this verification.

File: src/simulate.js [hunk 7]
```diff
@@ -17,18 +17,27 @@ var simulate = function (opts) {
...
     var simulator = new Simulator(opts);
     var showBrowser = opts.showbrowser;
+    var chromiumPath = opts.chromiumpath;
 
     if (!showBrowser) {
         var noBrowserMessage = 'The argument `showbrowser` is set to false. Please load simulated application in browser manually if needed.';
         log.warning(noBrowserMessage);
     }
 
+    if(process.platform === 'win32' && target === 'Chromium' && !chromiumPath){
```
Comment (EzioLi01) [inline src/simulate.js:27]: Can we support both upper and lower case? Or only lower case.

File: src/simulate.js [hunk 8]
```diff
@@ -17,18 +17,27 @@ var simulate = function (opts) {
...
     var simulator = new Simulator(opts);
     var showBrowser = opts.showbrowser;
+    var chromiumPath = opts.chromiumpath;
 
     if (!showBrowser) {
         var noBrowserMessage = 'The argument `showbrowser` is set to false. Please load simulated application in browser manually if needed.';
         log.warning(noBrowserMessage);
     }
 
+    if(process.platform === 'win32' && (target === 'chromium' || target === 'Chromium') && !chromiumPath){
```
Comment (EzioLi01) [inline src/simulate.js:27]: You can just use like `target = target.toLowerCase()` and verify.

Comment (RodBerw) [reply src/simulate.js]: I remember trying this at first, but it was not working for some reason at time, maybe I had altered the target var type while testing it before. I tryied it again and seems to work fine, thanks!

File: src/browsers/browser.js [hunk 9]
```diff
@@ -62,7 +63,7 @@ function launchBrowser(opts) {
     }
 }
 
-function getBrowser(target, dataDir, url) {
+function getBrowser(target, dataDir, chromiumPath, url) {
```
Comment (EzioLi01) [inline src/browsers/browser.js:66]: Please move `chromiumPath` to the last argument to avoid some unit test failures.

**Comment by EzioLi01** [comment]
LGTM, thanks for the contribution!

**Pull request merged**